spec/factories/album.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :album do
+    title { Faker::Music.album }
+    genre { Faker::Music.genre }
+    release_date { Faker::Date.between(from: '1950-01-01', to: Date.today) }
+
+    association :artist
```

Comment:
Você não precisa explicitar a associação. Quando voce define um método que não existe e há outro factory com o mesmo nome do método(nesse caso artist), este factory será usado como a associação ref: https://thoughtbot.github.io/factory_bot/ref/method_missing.html
```suggestion
    artist
```